.github/workflows/test.yml

```diff
@@ -21,12 +21,33 @@ jobs:
         server:
         - mir_kiosk
         - ubuntu_frame
-        - confined_shell
         - mir_test_tools
-        - mir_demo_server
-        - gnome_shell
+        channel: [stable]
         include:
+        - server: ubuntu_frame
+          channel: 24/edge
+
+        - server: mir_test_tools
+          channel: 24/edge
+
+        - server: mir_demo_server
+          channel: null
+
+        - server: mir_demo_server
+          channel: null
+          ppa: mir-team/dev
```

Comment:
Why have `mir_demo_server` listed twice with and without a ppa? I assume the one without a ppa uses the stable version and the one with uses the most recent one?

Reply:
Exactly, yes.